tools/build_defs/repo/http.bzl

```diff
@@ -216,7 +216,10 @@ to specify alternative URLs to fetch from.
             """A list of URLs to a file that will be made available to Bazel.
 
 Each entry must be a file, http or https URL. Redirections are followed.
-Authentication is not supported.""",
+Authentication is not supported.
+
+The resolution of this value is read top to bottom, and will fall through if a given
```

Comment:
Could this be reworded to something like "The list of URLs provided are evaluated in order, and an error could occur if a given URL fails to fetch the artifact."

Of course, feel free to add more emphasis or a better explanation if I didn't get the meaning quite right in the above phrase, but something like this would be a better explanatory sentence.

Reply:
The idea is that you do not get an error if one fails. You just try the next. You only get an error if ALL of them fail.
It could be better worded though, how about

URLs are tried in order until one succeeds, so you should list local mirrors first. If  all downloads fail the rule will fail.

Reply:
+1, that sounds good.

Reply:
Thanks, that's a clear way of putting it. Let me modify!